tests/swig/python/family/driver.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Nov 12 18:09:46 2017
+@author: andres
```

Comment:
Please add standard license header and change author to `The Souffle Developers` if possible.